go.mod

```diff
@@ -10,27 +10,37 @@ replace github.com/docker/docker => github.com/docker/engine v1.4.2-0.2019071716
 
 replace github.com/gomarkdown/markdown => github.com/status-im/markdown v0.0.0-20191113114344-af599402d015
 
-replace github.com/status-im/status-go/eth-node => github.com/status-im/status-go/eth-node v0.36.0
+replace github.com/status-im/status-go/eth-node => github.com/status-im/status-go/eth-node v1.0.1-0.20191219160300-4bee86b7e8da
```

Comment:
Let's release (create a tag similar to [eth-node/v1.0.0](https://github.com/status-im/status-go/releases/tag/eth-node%2Fv1.0.0)) instead of creating this replace. The same you could achieve using `go get github.com/status-im/status-go/eth-node@<commit_hash>` but having a release would be more clear.

Reply:
Done